README.md

```diff
@@ -318,6 +319,17 @@ disabled, existing imports are not rearranged, and new imports are always added
 sortImports: false
 ```
 
+### `appendImports`
+
+By default, ImportJS will add new imports below the existing imports.
```

Comment:
This is only true when `sortImports` and `groupImports` are false. I think the appending is fine to do without configuration so you can remove these docs and just change directly in the code. Sorry if I'm complicating things here.